src/Akka.Persistence.Sql/persistence.conf

```diff
@@ -99,7 +99,7 @@
       # should corresponding journal table be initialized automatically
       # if delete-compatibility-mode is true, both tables are created
       # if delete-compatibility-mode is false, only journal table will be created.
-      auto-initialize = off
+      auto-initialize = true
```

Comment:
Persistence plugins should work as is out of the box, database tables should be initialized automatically.
This is open for discussion, should this be reverted?